autoPyTorch/api/base_task.py

```diff
@@ -178,6 +180,14 @@ def __init__(
 
         self.stop_logging_server = None  # type: Optional[multiprocessing.synchronize.Event]
 
+        self.search_space_updates = search_space_updates
+        if search_space_updates is not None:
+            assert isinstance(self.search_space_updates,
```

Comment:
Now this should be a ValueError... In case the user or someone wants to catch this